src/ui/public/stringify/types/boolean.js

```diff
@@ -0,0 +1,41 @@
+import IndexPatternsFieldFormatProvider from 'ui/index_patterns/_field_format/field_format';
+
+export default function TruncateFormatProvider(Private) {
+
+  let FieldFormat = Private(IndexPatternsFieldFormatProvider);
+
+  class Bool extends FieldFormat {
+
+    constructor(params) {
+      super(params);
+    }
+
+    _convert(value) {
+
+      if (typeof value === 'string') {
+        value = value.trim().toLowerCase();
+      }
+
+      switch (value) {
+        case false:
+        case 0:
+        case 'false':
+        case 'no':
+          return 'false';
+        case true:
+        case 1:
+        case 'true':
+        case 'yes':
+          return 'true';
+        default:
+          return value.toString();
```

Comment:
can we use [_.asPrettyString](https://github.com/elastic/kibana/blob/13bdc523a576be33e90648b18e08401c161555d1/src/ui/public/utils/lodash-mixins/string.js#L12) instead?  the benefit being writing  - to empty fields